stack/stack.py

```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+    
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,5 +1,5 @@
 class Stack(object):
-    
+
     def __init__(self):
         self._list = []
 
```